ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -156,7 +158,7 @@ public void healtCheckRequestTest()
     Queue<HttpObject> responseParts = nettyClient.sendRequest(httpRequest, null, null).get();
     HttpResponse response = (HttpResponse) responseParts.poll();
     assertEquals("Unexpected response status", HttpResponseStatus.OK, response.getStatus());
-    ByteBuffer content = getContent(response, responseParts);
+    ByteBuffer content = getContent(responseParts, HttpHeaders.getContentLength(response));
```

Comment:
what if the content-length isn't set? What does `HttpHeaders.getContentLength(response)` return?

Particularly, why are these lines removed?
```
long contentLength = HttpHeaders.getContentLength(response, -1);
if (contentLength == -1) {		
  contentLength = HttpHeaders.getIntHeader(response, RestUtils.Headers.BLOB_SIZE, 0);		
}
ByteBuffer buffer = ByteBuffer.allocate((int) contentLength);
```